asp-server-core/src/main/java/de/jcup/asp/server/core/CoreAspServer.java

```diff
@@ -98,7 +98,7 @@ private void waitForClient() throws Exception {
         LOG.debug("Server waiting for client call");
         try (Socket clientSocket = serverSocket.accept();
                 PrintWriter out = new PrintWriter(clientSocket.getOutputStream(), true);
-                BufferedReader in = new BufferedReader(new InputStreamReader(clientSocket.getInputStream()))) {
+                BufferedReader in = new BufferedReader(new InputStreamReader(clientSocket.getInputStream(), "UTF-8"))) {
```

Comment:
Very fine. I completely missed that. Thank you.

_Remark for myself : On client side this must be UTF-8 too._